test_site/pages/dynamic_page.rb

```diff
@@ -2,5 +2,5 @@
 
 class TestDynamicPage < SitePrism::Page
   set_url '/dynamic{/letter}.htm'
-  set_url_matcher(/dynamic\/[ab]\.htm$/)
+  set_url_matcher(%r{dynamic\/[ab]\.htm$})
```

Comment:
I disagree with this cop. I actually think `//` looks better as it is what is used in many things. Is there a reason rubocop is enforcing this>?